README.md

```diff
@@ -1,19 +1,71 @@
-
-**The results of our optimizations are optimal given the simplifications we commented on above.** 
\ No newline at end of file
+- We don't know where your nodes or the traffic are located, which significantly impacts results. We cannot advise on the deployment locations of your chain nodes.
+- The optimization is **domain-based**; it provides optimal deployment for the **current** network, not a balanced network.
+- Given the dynamic nature of the network and other participants' reactions to your changes, be prepared to adjust your stakes periodically.
+- We expect network state convergence if enough node runners use this tool, but **convergence is not guaranteed**.
+- Recommendations may oscillate (i.e., a proposed strategy leads to a previous stake strategy). We don't expect this, but it could happen due to adversarial strategies among node runners.
+- By default, the tool considers relay averages over the past 24 hours. The chosen period for calculating average relays greatly affects outcomes. For example, if a chain had no relays in the last 24 hours, it won't be considered during optimization (no relays = no rewards).
+
+**The optimization results are optimal given the aforementioned simplifications.**
+
+### Requirements
+
+1. POKTscan API Token. Get yours at: [POKTscan API Token](https://poktscan.com/teams?tab=api_token)
+
+### How To Use
+
+1. Create a copy of `config.json.sample` and name it `config.json`.
+2. Edit `config.json` with your desired values.
+3. Generate the GraphQL types by running: `go generate`.
+4. Start the service with Docker: `docker compose up --build`. Exists a docker image build with our CI available at https://hub.docker.com/repository/docker/poktscan/wtsc/general
+
+If you want/need to modify the path and name of the config file, please use `CONFIG_FILE` to override the default `./config.json`
+
+### Understanding `config.json` Values
+
+```json
+{
+  "dry_mode": true, // If true, calls to "What to Stake" will write results to `results_path` or print them if `results_path` is empty.
+  "poktscan_api": "https://api.poktscan.com/poktscan/api/graphql", // POKTscan API endpoint
+  "poktscan_api_token": "", // POKTscan API token
+  "network_id": "testnet", // Can be "mainnet" or "testnet"
+  "tx_memo": "wtsc", // Transaction memo
+  "tx_fee": 10000, // Transaction fee
+  "domain": "example.com", // Your nodes domain (e.g., "poktscan.cloud" or "c0d3r.org")
+  "chain_pool": ["0021", "0003"], // Chain IDs available in your fleet
+  "servicer_keys": ["PRIVATE_KEY"], // List of private keys for signing stake transactions
+  "stake_weight": 4, // Used by the "What to Stake" service to estimate potential rewards (1-4)
+  "min_increase_percent": 5, // Minimum percentage increase expected to process stakes
+  "min_service_stake": [ // Minimum number of nodes for specific services
+    {
+      "service": "0021",
+      "min_node": 1
+    }
+  ],
```

Comment:
`chain_pool` should not be named `service_pool`? Because in the code, chain is always named service and I think is better to just have one name for the same thing.